collect_app/src/main/java/org/odk/collect/android/views/SlidingTabLayout.java

```diff
@@ -203,7 +204,14 @@ private void populateTabStrip() {
             }
 
             if (mTitleFontColor != -1) {
-                tabTitleView.setTextColor(getResources().getColor(mTitleFontColor));
+
+                int tabTitleTextColor;
```

Comment:
I am extremely sorry for the inconvenience it just did'nt strike me to use that class.I ll send a fix ASAP.